### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by prototype464** [GeorgeV220/VoteRewards-5.2.2_SNAPSHOT#11]
**Describe the bug**
NullPointerException

**To Reproduce**
Have a player join the server

**Expected behavior**
Not have an NPE error fill the console lol

**Server Info (please complete the following information):**
 - OS: Windows 10 Pro (Server is being locally run)
 - Java Version: AdoptOpenJDK jdk-16.0.1.9
 - Minecraft Server: Airplane #55 1.17.1

**Additional Context**
It's worth noting that the server is being connected to from a Velocity 3.0.0 proxy, running this setup locally as I'm creating a network for the first time from scratch and friends are helping build stuff.

It should also be noted that I have not configured VoteRewards, or really any plugin yet on this instance for that matter.

Here is the error log:
```
[12:51:34 WARN]: java.lang.NullPointerException
[12:51:34 WARN]:        at java.base/java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011)
[12:51:34 WARN]:        at java.base/java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006)
[12:51:34 WARN]:        at com.georgev22.voterewards.externals.utilities.maps.ConcurrentObjectMap.append(ConcurrentObjectMap.java:43)
[12:51:34 WARN]:        at com.georgev22.voterewards.utilities.player.UserVoteData$FileUserUtils$1.onSuccess(UserVoteData.java:697)
[12:51:34 WARN]:        at com.georgev22.voterewards.utilities.player.UserVoteData$FileUserUtils.setupUser(UserVoteData.java:766)
[12:51:34 WARN]:        at com.georgev22.voterewards.utilities.player.UserVoteData$FileUserUtils.load(UserVoteData.java:693)
[12:51:34 WARN]:        at com.georgev22.voterewards.utilities.player.UserVoteData.load(UserVoteData.java:280)
[12:51:34 WARN]:        at com.georgev22.voterewards.listeners.PlayerListeners.onJoin(PlayerListeners.java:48)
[12:51:34 WARN]:        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor789.execute(Unknown Source)
[12:51:34 WARN]:        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69)
[12:51:34 WARN]:        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76)
[12:51:34 WARN]:        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[12:51:34 WARN]:        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624)
[12:51:34 WARN]:        at net.minecraft.server.players.PlayerList.postChunkLoadJoin(PlayerList.java:360)
[12:51:34 WARN]:        at net.minecraft.server.players.PlayerList.lambda$placeNewPlayer$1(PlayerList.java:302)
[12:51:34 WARN]:        at net.minecraft.server.network.PlayerConnection.tick(PlayerConnection.java:307)
[12:51:34 WARN]:        at net.minecraft.network.NetworkManager.a(NetworkManager.java:555)
[12:51:34 WARN]:        at net.minecraft.server.network.ServerConnection.c(ServerConnection.java:196)
[12:51:34 WARN]:        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1643)
[12:51:34 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:482)
[12:51:34 WARN]:        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1465)
[12:51:34 WARN]:        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1261)
[12:51:34 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[12:51:34 WARN]:        at **java.base/java.lang.Thread.run(Thread.java:831)**
```

**Comment by GeorgeV220** [comment]
Hey, thank you for this bug report!! I'll look into it!!

Sorry for the late reply

**Comment by GeorgeV220** [comment]
Hello, the issue is probably fixed on the commit 08c711eb0f656258d16fa030c0f517d86bd705dd 
Please delete all user data from voterewards and try this version of the plugin from here:
https://jenkins.georgev22.com/job/VoteRewards/59/artifact/target/voterewards-v5.0.1-b59.jar
